src/main/java/lotto/domain/LottoRandom.java

```diff
@@ -0,0 +1,31 @@
+package lotto.domain;
+
+import static camp.nextstep.edu.missionutils.Randoms.pickUniqueNumbersInRange;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoRandom {
+
+    private List<Lotto> randomLotto;
+
+    public List<Lotto> generateLottoRandom(int lottoCounts) {
+        randomLotto = new ArrayList<>();
+
+        for (int i = 0; i < lottoCounts; i++) {
+            randomLotto.add(new Lotto(sortNumbers()));
+        }
+
+        return randomLotto;
+    }
+
+    private List<Integer> sortNumbers() {
+        return generateRandomNumber().stream().sorted().collect(Collectors.toList());
+    }
```

Comment:
메서드 체인이 길어지면 가독성이 떨어질 수 있어서 각 메서드를 줄바꿈하여 한 줄에 하나의 메서드 체인만 있도록 작성하는 것이 좋을 것 같습니다! https://github.com/JunHoPark93/google-java-styleguide
```suggestion
return generateRandomNumber().stream()
       .sorted()
       .collect(Collectors.toList());
```